.github/workflows/preview.yaml

```
@@ -29,82 +147,71 @@ jobs:
 
-      - name: Set up Node.js
-        uses: actions/setup-node@v4
+      - name: Download the latest snapshot artifact
+        uses: actions/download-artifact@v4
         with:
-          node-version: 18
-          cache: npm
-
-      - name: Install dependencies
-        run: npm ci
+          artifact-id: ${{ env.artifact_id }}
```

actionlint (1.7.4)

input "artifact-id" is not defined in action "actions/download-artifact@v4". available inputs are "github-token", "merge-multiple", "name", "path", "pattern", "repository", "run-id"

(action)